src/control/server/storage/scm/ipmctl.go

```diff
@@ -573,20 +620,39 @@ func (cr *cmdRunner) createDualNamespacesPerNUMA() (storage.ScmNamespaces, error
 	// For each region, create two namespaces each with half the total free capacity.
 	// Sanity check alignment.
 
-	regionFreeBytes, state, err := cr.getRegionState()
+	out, err := cr.showRegions()
 	if err != nil {
-		return nil, errors.WithMessage(err, "getting state")
+		return nil, errors.WithMessage(err, "show regions cmd")
 	}
-	if state == storage.ScmStateNoFreeCapacity {
-		return nil, errors.New("attempting to create namespaces but no free capacity")
+	if strings.Contains(out, outScmNoRegions) {
+		return nil, errors.New("no regions found")
+	}
+	regionFreeBytes, err := parseShowRegionOutput(cr.log, out)
+	if err != nil {
+		return nil, err
 	}
```

Comment:
Any reason not to use `getRegionFreeSpace` here?

Reply:
thanks, forgot to switch that over